### DISCUSSION TITLE
Per client configuration for access_token_strategy

### DISCUSSION THREAD

**Discussion by Tolsee** [ory/hydra#3479] [Q&A] [answered]
Hi, first of all, thank you very much for this wonderful piece of software 🙌🏼.

I have couple of questions regarding per-client `access_token_strategy`. I would appreciate if you could help me with it.

### Context
I came across [this guide](https://www.ory.sh/docs/oauth2-oidc/jwt-access-token#per-client-configuration) to setup `jwt` access token strategy for one of the client (I am aware that `jwt` is not recommended, that is the reason I only want to enable it for one-off client and keep `opaque` for others). But, this does not seem to be working in hydra's latest `v2.0.3`. (Is this guide meant for hydra's managed service 🤔?)

Digging deeper I came across this [pull request](https://github.com/ory/hydra/pull/3446), which seems to have added this feature. But, this is not part of `v2.0.3`. In the [changelog](https://github.com/ory/hydra/blob/master/CHANGELOG.md), I see below in [Features](https://github.com/ory/hydra/blob/master/CHANGELOG.md#features) of [0.0.0](https://github.com/ory/hydra/blob/master/CHANGELOG.md#000-2023-03-27) (which I suppose is will be the new release.)

> Allow setting access token type in client (https://github.com/ory/hydra/pull/3446) ([a6beed4](https://github.com/ory/hydra/commit/a6beed4659febd0917379d6da1e51d8ef75bc859))

### Questions

1. Is my understand correct that [pull request](https://github.com/ory/hydra/pull/3446) enables per-client `access_token_strategy` in self-hosted ory-hydra as described in [this guide](https://www.ory.sh/docs/oauth2-oidc/jwt-access-token#per-client-configuration)?
2. Can you share if any timeline for this [new release](https://github.com/ory/hydra/blob/master/CHANGELOG.md#000-2023-03-27)?

Thank you in advance.

**Comment by snnay22** [comment]
We have the same issue. It will really helpful if we can please get a tentative time line of the release as mentioned above. Thanks in advance.

**Comment by Tolsee** [comment, marked as answer]
As we have new release https://github.com/ory/hydra/releases/tag/v2.1.0 already. We can close this discussion.
